docs/en/about-appium/getting-started.md

```diff
@@ -251,7 +251,7 @@ We've only scratched the surface of what you can do with Appium. Check out
 these resources to help you on your journey:
 
 - The Appium [command reference](https://appium.io/docs/en/commands/status/) - learn about what commands are available, how to use them with specific client libraries, etc...
-- The [sample-code](https://github.com/appium/appium/sample-code) repository, where lots more code samples are available
+- The [sample-code](https://github.com/appium/sample-code) repository, where lots more code samples are available
```

Comment:
https://github.com/appium/appium/tree/master/sample-code

Reply:
Right, the update sample-code